Comment-Code-Poltergeists/nutshell
# Rb display chat messages

# Description

Display a chronological list of chat messages from all users who are your friend as well as yourself. Added in the standard grey-background style to match other elements. 

Fixes #14 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

`git fetch --all`
`git checkout rb-display-chatMessages`
On page load, check that all of your/your friends' chat messages are showing in the 'MESSAGES' window and that box is already scrolled to the bottom automatically. The messages should be gray and also have the userName and timestamp above each of them.

# Checklist:
